### PR TITLE
Define categories for json exports

### PR DESCRIPTION
**Pull request by dpordomingo**
The most relevant change is that once approved, the posts can be categorized by scope;
current categories are **technical**, science, programming and **culture** (for non-technical posts)
The system will accept many different categories for the same post; those proposed or any other.

#### Does it have any implication:
- Blog -> no implications, there is no category shown in the blog at this moment nor after approving the PR
- WEB -> YES. The new landing HP will show (separately) the first "n" posts from "**technical**" and "**culture**" categories

#### As a blog post writter, should you know something?
Yes. Maybe some posts should appear in the Landing, and other posts should not; In that case, only certain posts should be categorized with "**technical**" or "**culture**" ; It should be confirmed by the PM in order to have a guideline about it; anyway, it is not related with the approval of this PR :D

#### What about other categories as **science** or **programming**?
They are not used currently, but it could be a good idea to use them in case of further use

#### How will be those categories exposed?
Once approved the PR, the allowed categories will be exposed under 
http://blog.sourced.tech/json/CATEGORY_NAME

### CODE REVIEW

File: content/post/100-awesome-women-in-the-open-source-community-you-should-know.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
---- 
+---
 author: eiso 
 date: 2016-05-25 
 title: "100 Awesome Women In The Open-Source Community You Should Know" 
 draft: false 
 image: /post/100-awesome-women-in-the-open-source-community-you-should-know/intro.jpg 
 description: "100 Awesome Women In The Open-Source Community You Should Know" 
---- 
+categories: ["business"]
```
Comment (mcuadros) [inline content/post/100-awesome-women-in-the-open-source-community-you-should-know.md]: business?

File: content/post/dot_go_2015.md [hunk 2]
```diff
@@ -4,6 +4,7 @@ date: 2015-11-13
 title: "dotGo2015: Undelivered Expectations"
 image: /post/dot_go_2015/intro.jpg
 description: "dotGo 2015, an excellent venue, not-so-techy talks."
+categories: ["business"]
```
Comment (mcuadros) [inline content/post/dot_go_2015.md]: business?

File: content/post/retreat-story.md [hunk 3]
```diff
@@ -4,6 +4,7 @@ date: 2015-10-12
 title: Taking Coding to the Mountains
 image: /post/retreat-story/9239.jpg
 description: "The Tyba engineers enjoyed a wonderful experience these last days of summer: spending two days in a country house, far away from the madding crowds, to participate in an Open Source Hackaton."
+categories: ["business"]
```
Comment (mcuadros) [inline content/post/retreat-story.md]: clearly we need to agree what is business

Comment (ajnavarro) [inline content/post/retreat-story.md]: **business** to **other** maybe?

**Comment by dpordomingo** [comment]
as already talked, "culture" term could be a proper one to define non-technical section ("our journey" landing section)

File: content/post/reading_pyspark_pickles_locally.md [hunk 4]
```diff
@@ -5,6 +5,7 @@ title: "Reading PySpark pickles locally"
 draft: false
 image: /post/reading_pyspark_pickles_locally/intro.jpg
 description: "How to load Hadoop SequenceFile-s with Python serialized objects without having to install Spark - using src-d/sparkpickle"
+categories: ["science", "technical"]
```
Comment (vmarkovtsev) [inline content/post/reading_pyspark_pickles_locally.md]: It is not science for sure

Comment (dpordomingo) [inline content/post/reading_pyspark_pickles_locally.md]: done

File: content/post/100-awesome-women-in-the-open-source-community-you-should-know.md (hunk 1 above)
Comment (dpordomingo) [inline content/post/100-awesome-women-in-the-open-source-community-you-should-know.md]: done

File: content/post/dot_go_2015.md (hunk 2 above)
Comment (dpordomingo) [inline content/post/dot_go_2015.md]: done

**Pull request merged**